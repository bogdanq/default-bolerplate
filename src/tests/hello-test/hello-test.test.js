import { render } from "@testing-library/react"
import React from "react"

import { HelloTest } from "./index"

describe("HelloTest component", () => {
  it('should render without "name" props', () => {
    const { getByText } = render(<HelloTest />)

    expect(getByText("Hey")).toBeDefined()
  })

  it('should render with "name" props', () => {
    const { getByText } = render(<HelloTest name="Test" />)

    expect(getByText("Hello, Test")).toBeDefined()
  })

  it("snapshot", () => {
    const { container } = render(<HelloTest name="Test" />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
