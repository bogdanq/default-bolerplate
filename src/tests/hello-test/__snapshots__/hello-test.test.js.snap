// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloTest component snapshot 1`] = `
<h1
  data-testid="h1"
>
  Hello, 
  Test
</h1>
`;
